packages/ui-causal-graph-editor/src/types.tsx

```diff
@@ -205,6 +205,14 @@ export interface SimulationAttributes {
 }
 export type SimulationGraph = AbstractGraph<SimulationNode, SimulationEdge, SimulationAttributes>;
 
+// Defines the Layering algorithms supported by PlanarLayout
+export enum LayeringAlgorithm {
```

Comment:
Since this is defined in types now do you think we should call it PlanarLayeringAlgortithm or something more specific to make it clear it only works for that?